src/o365/spo/commands/list/list-label-set.ts

```diff
@@ -0,0 +1,241 @@
+          headers: Utils.getRequestHeaders({
+            authorization: `Bearer ${siteAccessToken}`,
+            'accept': 'application/json;odata=nometadata'
+          }),
+          json: true
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request...');
+          cmd.log(requestOptions);
+          cmd.log('');
+        }
+
+        return request.get(requestOptions)
+        .then((listInstance: ListInstance): Promise<string> => {
+          
+          if (this.debug) {
+            cmd.log('Response:');
+            cmd.log(listInstance);
+            cmd.log('');
+          }
+
+          return Promise.resolve(listInstance.RootFolder.ServerRelativeUrl);
+        });
+      })
+      .then((listServerRelativeUrl: string): request.RequestPromise => {
+
+        const listAbsoluteUrl: string = Utils.getAbsoluteUrl(args.options.webUrl, listServerRelativeUrl);
+        const requestUrl: string = `${args.options.webUrl}/_api/SP_CompliancePolicy_SPPolicyStoreProxy_SetListComplianceTag`;
+        const requestOptions: any = {
+          url: requestUrl,
+          headers: Utils.getRequestHeaders({
+            authorization: `Bearer ${siteAccessToken}`,
+            'accept': 'application/json;odata=nometadata'
+          }),
+          body: {
+            listUrl: listAbsoluteUrl,
+            complianceTagValue: args.options.label,
+            blockDelete: args.options.blockDelete || false,
+            blockEdit: args.options.blockEdit || false,
+            syncToItems: args.options.syncToItems || false
+          },
+          json: true
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request...');
+          cmd.log(requestOptions);
+          cmd.log('');
+        }
+
+        return request.post(requestOptions);
+      })
+      .then((res: any): void => {
+
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-u, --webUrl <webUrl>',
+        description: 'The URL of the site where the list is located'
+      },
+      {
+        option: '--label <label>',
+        description: 'The label to set on the list'
+      },
+      {
+        option: '-t, --listTitle [listTitle]',
+        description: 'Title of the list where the field is located. Specify only one of listTitle, listId or listUrl'
+      },
+      {
+        option: '-l, --listId [listId]',
+        description: 'ID of the list where the field is located. Specify only one of listTitle, listId or listUrl'
+      },
+      {
+        option: '--listUrl [listUrl]',
+        description: 'Server- or web-relative URL of the list where the field is located. Specify only one of listTitle, listId or listUrl'
+      },
+      {
+        option: '--syncToItems [syncToItems]',
```

Comment:
This option is meant to be used as a switch. As such, it's sufficient to use just `--syncToItems` without `[syncToItems]`. `[syncToItems]` indicates that a value should be passed, eg. `--syncToItems YesDoIt`